src/hotspot/share/prims/jvmtiImpl.cpp

```diff
@@ -301,8 +204,8 @@ void VM_ChangeBreakpoints::doit() {
 // a JVMTI internal collection of JvmtiBreakpoint
 //
 
-JvmtiBreakpoints::JvmtiBreakpoints(void listener_fun(void *,address *)) {
-  _bps.initialize(this,listener_fun);
+JvmtiBreakpoints::JvmtiBreakpoints()
+    : _elements(5, mtServiceability) {
```

Comment:
Do we have tests that create more than 5 breakpoints? I just want to make sure the array growing code is exercised. You could stress it by testing with the initial size set to 1.

Reply:
This is the same we had before (in `GrowableCache::initialize`, old line 142):
`_elements       = new (mtServiceability) GrowableArray<GrowableElement*>(5, mtServiceability);`
I set the initial size to 1,  JDI tests (some of them have more than 1 breakpoint) passed